HW9/Homework9.cpp

```diff
@@ -0,0 +1,152 @@
+#include <iostream>
+#include <cstring>
+
+
+// Task 1
+
+void TranslateArray(int Numbers[], int Size)
+{
+	for (int i = 0; i < Size / sizeof(int); i++)
+	{
+		if (Numbers[i] > 0)
+		{
+			Numbers[i] *= 2;
+		}
+		else
+		{
+			Numbers[i] = 0;
+		}
+	}
+}
+
+// Task 2
+
+void ToUppercase(char Str[])
+{
+	for (int i = 0;  ; i++)
+	{
+		if (Str[i] == '\0')
+		{
+			break;
+		}
+		if (Str[i] >= 97 && Str[i] <= 122)
+		{
+			std::cout << Str[i];
+			Str[i] -= 32;
+
+		}
+	}
+}
+
+// Task 3
+bool IsPalindrom(const char Str[])
+{
+	for (int ArraySize = 0;; ArraySize++)
+	{
+		if (Str[ArraySize] == '\0')
```

Comment:
split this logic in two parts, not one inside of the other.
1. Find length
2. Execute comparison algorithm